_includes/head.html

```diff
@@ -10,6 +10,8 @@
 
   <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,800,400italic,300,300italic,700italic,800italic' rel='stylesheet' type='text/css'>
 
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Can you fetch only the required fonts? Maybe even svgs from somewhere is fine. Let's not fetch the entire font-awesome library here